src/loaders/features/features.js

```diff
@@ -25,9 +25,8 @@ export const featurePriority = {
   [FEATURE_NAMES.spa]: 5,
   [FEATURE_NAMES.ajax]: 6,
   [FEATURE_NAMES.sessionTrace]: 7,
-  [FEATURE_NAMES.pageAction]: 8,
-  [FEATURE_NAMES.softNav]: 9,
-  [FEATURE_NAMES.sessionReplay]: 10,
-  [FEATURE_NAMES.logging]: 11,
-  [FEATURE_NAMES.genericEvents]: 12
+  [FEATURE_NAMES.softNav]: 8,
+  [FEATURE_NAMES.sessionReplay]: 9,
+  [FEATURE_NAMES.logging]: 10,
+  [FEATURE_NAMES.genericEvents]: 11
```

Comment:
We need to open a ticket to rip this ordering out and instead rely on `src/loaders/features/featureDependencies.js`